### PR TITLE
feat(css): add information on scope (at-rule) boundary behavior

### PR DESCRIPTION
**Pull request by bayesian-intelligence**
### Description

Add more details on @scope boundary behaviour.

### Motivation

The default inclusivity / exclusivity behaviour is non-obvious. There is description of it in the MDN page, and no example showing how to invert the selectivity if needed. There is also nothing that explains that a scope limit can be used with an inline @scope.

### Additional details

In this revision, the selectivity behaviour is described, and examples given of how to alter the behaviour if needed. 

The universal child selector approach is recommended in this WG bug for the lower bound: https://github.com/w3c/csswg-drafts/issues/6577 . It generalises to the upper bound.

### CODE REVIEW

File: files/en-us/web/css/@scope/index.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ The `@scope` at-rule contains one or more rulesets (termed **scoped style rules*
    }
    ```
 
+   The scope's upper bound is inclusive and its lower bound is exclusive. To change this behavior, you can combine either selector with a universal child selector. For example, `@scope (scope root) to (scope limit > *)` would make both bounds inclusive, `@scope (scope root > *) to (scope limit)` would make both bounds exclusive, while `@scope (scope root > *) to (scope limit > *)` would give an exclusive upper bound and an inclusive lower bound.
+
```
Comment (estelle) [inline files/en-us/web/css/@scope/index.md:27]: This is useful information

This would be better placed in the description. This is the syntax section. The child selector portion is useful, but not required syntax. Also, the upper and lower bound is not yet defined, so is hard to parse here, but may be much more understandable if it comes after "The `.article-body` scope root selector defines the upper bound of the DOM tree scope in which the ruleset will be applied, and the figure scope limit selector defines the _lower bound_."

**Comment by bsmth** [comment]
Hi @bayesian-intelligence, there's some reviewer feedback above. Do you plan on coming back to this one? Thanks :)

**Comment by bayesian-intelligence** [comment]
I'm no longer on break, so harder for me to get back to. : )

I'm happy for this to go wherever it needs to. I placed it in the syntax section because it was originally a syntax discussion (with 'inclusive'/'exclusive' literals, among other proposals), and a note in https://github.com/w3c/csswg-drafts/issues/6577 that the explicit syntax would be dropped, but that documentation (in the spec) should describe the convention instead (i.e., "RESOLVED: make exclusive lower boundaries the default and **add an example to the spec on how to do inclusive**".)

(For me, this was also question #1 on limit behaviour before I could rely on it.)

**Review by estelle — APPROVED** [review]
Thank you! 
Also, congratulations on your first merged MDN content PR. 
Welcome to the team!
